olive/data/container/dummy_data_container.py

```diff
@@ -32,3 +38,96 @@ class DummyDataContainer(DataContainer):
         DataComponentType.LOAD_DATASET.value: "dummy_dataset",
         DataComponentType.DATALOADER.value: "skip_dataloader",
     }
+
+
+@Registry.register(DataContainerType.DATA_CONTAINER)
+class TransformersDummyDataContainer(DummyDataContainer):
+    """Dummy data container for transformer model.
+
+    The way to create a dummy data container for transformer model:
+        dummy_data_config = DataConfig(
+            name="dummy",
+            type="TransformersDummyDataContainer",
+            "load_dataset_config"={
+                "params": {
+                    "batch_size": 1,
```

Comment:
Do you think it might be more consistent to have batch_size as parameter for `data_loader_config`? I tested it locally and torch dataloader is able to handle batching of the list of key-values correctly (although it turns the tuple into list but I think that's fine?). 

Otherwise, it feels weird to have `batch_size` as a parameter of load dataset.

Reply:
True. I agree with you. Will update it.

Reply:
Updated.